charts/netdata/README.md

```diff
@@ -1,14 +1,10 @@
-<!--
-title: "Netdata Helm chart for Kubernetes deployments"
-custom_edit_url: https://github.com/netdata/helmchart/edit/master/charts/netdata/README.md"
-sidebar_label: "Helm chart for Kubernetes"
-learn_status: "Published"
-learn_rel_path: "Configuration"
--->
-
 # Netdata Helm chart for Kubernetes deployments
 
-[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/netdata)](https://artifacthub.io/packages/search?repo=netdata) ![Version: 3.7.48](https://img.shields.io/badge/Version-3.7.48-informational) ![AppVersion: v1.38.1](https://img.shields.io/badge/AppVersion-v1.38.1-informational)
+<a href="https://artifacthub.io/packages/search?repo=netdata" target="_blank" rel="noopener noreferrer"><img loading="lazy" src="https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/netdata" alt="Artifact HUB" class="img_node_modules-@docusaurus-theme-classic-lib-theme-MDXComponents-Img-styles-module"></img></a>
+
+<img src="https://img.shields.io/badge/Version-3.7.48-informational" alt="Version: 3.7.48"></img>
+
+<img loading="lazy" src="https://img.shields.io/badge/AppVersion-v1.38.1-informational" alt="AppVersion: v1.38.1" class="img_node_modules-@docusaurus-theme-classic-lib-theme-MDXComponents-Img-styles-module"></img>
```

Comment:
Why separate lines?

I think it looks neater as it was

<img width="983" alt="Screenshot 2023-04-05 at 18 51 59" src="https://user-images.githubusercontent.com/22274335/230135439-91646839-5d8a-4818-980e-8ef890657b26.png">

Reply:
> make badges render properly both on GitHub and Learn

Can you show how it looks on learn?

Reply:
Yup,

In Learn it is like this:
![image](https://user-images.githubusercontent.com/70198089/230136663-0a3a79ed-57b1-45b6-aa69-c30179c4cc4b.png)


And now in my local deployment of this, it looks like this:
![image](https://user-images.githubusercontent.com/70198089/230136465-53ad9c1c-0a3d-4c62-a7b8-0013fd290e99.png)

My understanding is that docusaurus treats these as images, so it scrambles the arrangement.